narwhal/primary/src/synchronizer.rs

```diff
@@ -97,10 +98,10 @@ struct Inner {
     dag: Option<Arc<Dag>>,
     /// Contains Synchronizer specific metrics among other Primary metrics.
     metrics: Arc<PrimaryMetrics>,
-    /// Background tasks synchronizing worker batches for processed certificates.
-    batch_tasks: Mutex<JoinSet<DagResult<()>>>,
     /// Background tasks broadcasting newly formed certificates.
     certificate_senders: Mutex<JoinSet<()>>,
+    /// A background task that synchronizes batches
```

Comment:
Describe elements in the tuple? Also it is unnecessary to send the `certified` bool.